### ISSUE TITLE
Importing the generated .ovpn file into OpenVPN Connect mobile app fails

### ISSUE THREAD

**Issue by duckpuppy** [luxflux/puppet-openvpn#187]
The generated client .ovpn files have the following options set:
```
ca keys/<client-name>/ca.crt
cert keys/<client-name>/<client-name>.crt
key keys/<client-name>/<client-name>.key
```
Because the directives are present in the file, the mobile client will not import the .ovpn file due to the files not being present.  Simply removing those three lines from the generated .ovpn file allows it to import successfully since those keys are also contained inline in the .ovpn file itself.  If having those directives is necessary for some other reason, perhaps an option could be introduced to control the generation of those directives.

**Comment by luxflux** [comment]
Unfortunately I am not a user of the `.ovpn` files. Maybe @elisiano can give us some insights?

**Comment by elisiano** [comment]
Currently I'm using an older version of the module (prior to the ovpn refactoring). I'll try to update and report back. 
Although if what @duckpuppy says is true, we have an issue. There should not be any external file reference (that's the whole point of having .ovpn files).

**Comment by elisiano** [comment]
So yeah, this is the issue: https://github.com/luxflux/puppet-openvpn/blob/master/manifests/client.pp#L336

It includes the original config in the ovpn file which contains references to the external files.

**Comment by elisiano** [comment]
@duckpuppy did you have a chance to test the PR I submitted?

**Comment by duckpuppy** [comment]
I haven't seen that a PR was submitted for this ticket.  I should have some
time tomorrow to try it out.

**Comment by luxflux** [comment]
@duckpuppy did it work out?

**Comment by luxflux** [comment]
I assume this is fixed by #190.
